docs/docs-content/clusters/edge/site-deployment/deploy-custom-registries/webhook-disable.md

```diff
@@ -0,0 +1,425 @@
+flexibility to configure your own image pull behavior. This guide guides you through how to disable the Palette agent
+webhook and provides an example custom image pull configuration.
+
+## What Happens When You Disable the Webhook
+
+When the agent webhook is disabled, the Palette agent will not redirect any image pull operation by default. This means
+that even if you specify an external registry in the **user-data**, the Palette agent will not pull images from that
+registry unless it is otherwise configured to do so. This also means that the Palette agent will not pull images from
+the local Harbor registry, even if the images are downloaded and stored in the registry, unless it is otherwise
+configured to do so. Disabling the webhook removes restrictions, but does place the burden of ensuring that images are
+pulled from the correct locations on yourself.
+
+You may consider disabling the webhook if you want to configure your cluster to pull images from multiple authenticated
+registries, or you simply do not want the default behavior that forces image pulls to be redirected to the local Harbor
+registry. Once the webhook is disabled, you can then take advantage of the rewrite features of some Kubernetes
+distributions such as K3s and RKE2, or other redirect mechanism that you implement on your own to customize the image
+pull behavior.
+
+## Prerequisites
+
+- The process to disable webhook is based on the EdgeForge process. We recommend that you familiarize yourself with
+  [EdgeForge](../../edgeforge-workflow/edgeforge-workflow.md) and the process to build Edge artifacts.
+
+- A physical or virtual Linux machine with _AMD64_ (also known as _x86_64_) processor architecture to build the Edge
+  artifacts. You can issue the following command in the terminal to check your processor architecture.
+
+  ```bash
+  uname -m
+  ```
+
+- Minimum hardware configuration of the Linux machine:
+
+  - 4 CPU
+  - 8 GB memory
+  - 150 GB storage
+
+- [Git](https://git-scm.com/downloads). You can ensure git installation by issuing the `git --version` command.
+
+- [Docker Engine](https://docs.docker.com/engine/install/) version 18.09.x or later. You can use the `docker --version`
+  command to view the existing Docker version. You should have root-level or `sudo` privileges on your Linux machine to
+  create privileged containers.
+
+- A [Spectro Cloud](https://console.spectrocloud.com) account.
+
+## Procedure
+
+### Disable Webhook
+
+1. Clone the **CanvOS** repository.
+
+   ```shell
+   git clone https://github.com/spectrocloud/CanvOS.git
+   ```
+
+2. Change into the **CanvOS** directory.
+
+   ```shell
+   cd CanvOS
+   ```
+
+3. View the available tags and use the latest available tag. This guide uses `v4.5.0` as an example.
+
+   ```shell
+   git tag
+   git checkout v4.5.0
+   ```
+
+4. In your **user-data** file, set `stylus.imageRedirectWebhook.enable` to `false`. This parameter defaults to true if
```

Comment:
@lennessyy we should add this paramter definition to the [Installer Configuration page](https://deploy-preview-4098--docs-spectrocloud.netlify.app/clusters/edge/edge-configuration/installer-reference/).